src/cdk/index.ts

```diff
@@ -20,6 +22,13 @@ export interface OttofellerCDKProjectOptions
    * @default 0.0.1
    */
   readonly initialReleaseVersion?: string
+
+  /**
+   * Set up GraphQL dependencies and supplementary script.
+   *
+   * @default true
+   */
+  readonly isGraphqlEnabled?: boolean
```

Comment:
@quesabe let's be more verbose:
```suggestion
  readonly isGraphqlCodegenEnabled?: boolean
```